idd/Energy+.idd.in

```diff
@@ -19458,6 +19458,7 @@ RoomAir:Node:AirflowNetwork:InternalGains,
       \key ElectricLoadCenter:Inverter:LookUpTable
       \key ElectricLoadCenter:Storage:Battery
       \key ElectricLoadCenter:Storage:Simple
+      \key ElectricLoadCenter:Storage:Converter
```

Comment:
@EnergyArchmage Since we are past the IO lockdown, this has me concerned.  If this can't go in because of the IDD change here, is there anything you'd like to, or be able to, extract out as a separate, tiny, bug fix PR?  If not, this may have to wait until after release.  Let me get a feel from some interface folks here.

@kbenne @macumber Does this type of IDD change have an effect on you all?  This is a refactoring PR that preps the code for a separate bug fix PR (#5529) that would be nice to get in, but since it affects the idd here, I'm curious if this will have an effect on you downstream.